encrypted-media-respec.html

```diff
@@ -2089,15 +2080,12 @@
           </ol>
         </section>
 
-        <section id="session-close">
-          <h4>Session Close</h4>
-          <p>The Session Close algorithm is run when the CDM closes the session associated with a <a>MediaKeySession</a> object.</p>
+        <section id="close-session">
```

Comment:
I expected that `close()` would call this algorithm. Otherwise, I think it only applies to Close All Sessions. It seems those two things should be consistent (except where the CDM is unavailable).

However, see my comments at lines 1341 and 1667. I think we only need to perform these operations in `close()`, meaning we could eliminate this algorithm.